### ISSUE TITLE
Issue with handling cflags/cxxflags with spack load

### ISSUE THREAD

**Issue by pramodk** [spack/spack#2265]
As #2227 is merged, I am testing issues mentioned in #1953. `spack install` now works fine. I have installed package with 3 different flags:

```bash
$ spack find -fl
==> 3 installed packages.
-- linux-rhel6-x86_64 / gcc@4.9.0 -------------------------------
72luxlk myPkg@develop%gcc cflags="-O1"
joxds3t myPkg@develop%gcc cflags="-O1 -DLAYOUT=1"
3xc54nb myPkg@develop%gcc cflags="-O1 -g"
```

But if I now use same `flags` to load package, I see : 
```
$ spack load myPkg %gcc cflags=\"-O1 -g\"

usage: spack [-h] [-d] [-D] [-k] [-m] [-p] [-v] [-V] SUBCOMMAND ...
spack: error: unrecognized arguments: -g"
```

**Comment by becker33** [comment]
Is this issue resolved as of #2769?

**Comment by adamjstewart** [comment]
No response from @pramodk so I'm assuming this has been fixed, please reopen if this is not the case.
